### DISCUSSION TITLE
原站搭建在内网，使用公网服务器的nginx反代问题

### DISCUSSION THREAD

**Discussion by mgodex** [lsky-org/lsky-pro#481] [问答] [answered]
例如，原站内网访问为127.0.0.1:8080
使用ddns或带公网的服务器nginx反代后，生成图片的路径域名依然为127.0.0.1，如何修改？或者自动生成为当前访问的域名。

**Comment by mgodex** [comment, marked as answer]
问题解决，自己写了一篇文章，希望能帮助到各位：https://www.eg3.cn/posts/11.html

**Comment by yaoguaiba232** [comment]
https://www.eg3.cn/posts/11.html这个页面失效了，如何修改？或者自动生成为当前访问的域名。这个是要怎么修改呢。
